db/seeds/02_data.rb

```diff
@@ -34,7 +34,8 @@
   puts 'Deleting old versions'
   Version.delete_all
 
-  puts 'Loading CSVs. Why not do some calf raises while you wait? ... '
+  puts 'Loading CSVs. Why not do some calf raises while you wait? ...' 
+  SeedUtils.seed_tables_with_group('Accreditation', user)
```

Comment:
Why is this instance of the SeedUtils method called using a string instead of class-name?

Reply:
Accreditation isn't a class in this case so the string `'Accreditation'` is used to upload the excel file.